src/app/components/Copilot/Copilot.tsx

```diff
@@ -14,41 +14,45 @@ let GlobalConversationID: string;
 
 // going to use get all conversations with a few extra steps to store the current conversations locally.
 export function createNewConversation() {
+  try{
```

Comment:
Hey @OlibhiaGhosh! This is great thank you!

Small thing, could you ensure that the spacing for the `try/catch` is correct?

Ex:
```
try {
  // ...
} catch (error) {
  // ...
}
```

Reply:
I have checked the spacing. It seems correct to me.

Is there any issue?

Reply:
Just keeping the syntax the same throughout so that there is a space between `try` and `{` and for the `catch` as well 👍

Reply:
I have made the changes!

Can you please review it?